src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const initialState = { count: 0, books: [] };
```

Comment:
- [ ] Please, consider use an empty array as initialState. This will help to make your project more accordingly to the requirements. 🐶